akka-persistence-testkit/src/main/resources/reference.conf

```diff
@@ -0,0 +1,30 @@
+#################################################
+# Akka Persistence Testkit Reference Config File #
+#################################################
+
+# This is the reference config file that contains all the default settings.
+# Make your edits/overrides in your application.conf.
+
+akka.persistence.testkit {
+
+  # confugiration for persistence testkit for messages
+  messages {
+    # enable serialization of the persisted messages
+    serialize = true
+    # timeout for assertions
+    assert-timeout = 1s
```

Comment:
we use `3s` for `single-expect-default` so let's use that here also

Reply:
corrected